doc/developer-notes.md

```diff
@@ -1080,6 +1080,10 @@ A few guidelines for introducing and reviewing new RPC interfaces:
     until the wallet is caught up to the chainstate as of the RPC call's entry.
     This also makes the API much easier for RPC clients to reason about.
 
+- The RPC call `getbalances` also returns `m_last_block_processed` in the `lastblock`
```

Comment:
s/m_last_block_processed/lastblock/

Reply:
I don't understand, sorry. We are returning `m_last_block_processed` in the field `lastblock`.

Reply:
Sorry, I mean that we could avoid C++ variable names here. I was thinking something like
```
- The `getbalances` RPC now returns `lastblock` which is the wallet's last processed block
  hash at the time the balances were calculated.
```

Reply:
Thanks. Have update the docs.